app/presenters/migration/index.rb

```diff
@@ -50,6 +50,7 @@ def include_nav_searchbar
 
     def call_to_action
       t('call_to_action')
+      false
```

Comment:
Can't we just remove the entire method from this class instead?

Reply:
Yes.  I've removed it